src/GHC/TypeLits/Extra/Solver.hs

```diff
@@ -310,7 +310,10 @@ lookupExtraDefs = do
 #if MIN_VERSION_ghc(9,2,0)
     md2 <- lookupModule ordModule basePackage
 #endif
-    ExtraDefs <$> look md "Max"
+    ExtraDefs <$> pure typeNatAddTyCon
```

Comment:
I'd rather not add these `TyCon`s here, since they're technically not defined in `GHC.TypeLits.Extra`.